<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ffplayClass</class>
 <widget class="QMainWindow" name="ffplayClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ffplay</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelVideoView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenFile">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件</string>
    </property>
   </widget>
   <widget class="QWidget" name="ControlStatusBar" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>631</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(24, 24, 24, 255), stop:1 rgba(64, 64, 64, 255));</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionssss">
   <property name="text">
    <string>ssss</string>
   </property>
  </action>
  <action name="actioneee">
   <property name="text">
    <string>eee</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ffplay.qrc"/>
 </resources>
 <connections/>
</ui>
